docarray/array/storage/sqlite/seqlike.py

```diff
@@ -97,8 +97,3 @@ def __eq__(self, other):
             and type(self._config) is type(other._config)
             and self._config == other._config
         )
-
-    def __add__(self, other: Union['Document', Sequence['Document']]):
-        v = type(self)(self, storage='sqlite')
-        v.extend(other)
-        return v
```

Comment:
@alaeddine-13 @davidbp Is this revision is decided?

Reply:
For me `__add__` only makes sense for memory document array.